src/EnergyPlus/Furnaces.hh

```diff
@@ -676,6 +678,18 @@ namespace Furnaces {
 		Real64 & PartLoadRatio // coil part-load ratio
 	);
 
+	void
+	SetMinOATCompressor(
```

Comment:
@rraustad Why go through all this effort to pass the min OAT up to the parent object and check it there?  Why not just check it in the coil object and let it return with zero load, same as if the coil schedule was off for that timestep?